code/AssetLib/glTF2/glTF2Asset.h

```diff
@@ -1266,9 +1266,8 @@ class Asset {
     size_t mSceneLength;
     size_t mBodyOffset;
     size_t mBodyLength;
-    IdMap mUsedIds;
-    std::map<std::string, int, std::less<>> mUsedNamesMap;
     Ref<Buffer> mBodyBuffer;
+    std::map<std::string, int, std::less<>> lastUsedID;
```

Comment:
Why `map`? Why not `unordered_map`? Is iteration order necessary for some reason?

Reply:
>  Is iteration order necessary for some reason?

It should not be

> Why map? Why not unordered_map?

C++ is not my native language -- that's why. I code in [eC](https://ec-lang.org).

I can amend the PR to use `unordered_map` if you'd like or feel free to do it.

Reply:
Also because I followed the example of `mUsedNamesMap` being used previously, where an `unordered_map` would probably have been fine in that case as well.